src/server/Document/component.test.jsx

```diff
@@ -51,15 +51,15 @@ describe('Document Component', () => {
     );
 
   expect(
-    shallowDocument({
+    Document({
       service: 'news',
       isAmp: false,
       testName: 'should render correctly',
     }),
   ).toMatchSnapshot();
```

Comment:
do we need this here

Reply:
No as, it generates two `undefined` snapshots, which aren't supposed to exist.